### ISSUE TITLE
[None] is being written to credentials file

### ISSUE THREAD

**Issue by jbpratt** [cshamrick/stsauth#57]
**Describe the bug**
After selecting an account profile, credentials are requested then an exception is raised

It seems this happens when authenticating to an account that already exists in the credentials file, when deleting the `None`, I am able to authenticate, with it then writing `None`, and it does not happen to accounts I have never authenticated to before.

**Expected behavior**
Credentials are requested

```
Traceback (most recent call last):
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/configloader.py", line 149, in raw_config_parse
    cp.read([path])
  File "/usr/lib/python3.8/configparser.py", line 697, in read
    self._read(fp, filename)
  File "/usr/lib/python3.8/configparser.py", line 1067, in _read
    raise DuplicateSectionError(sectname, fpname,
configparser.DuplicateSectionError: While reading from '/home/jbpratt/.aws/credentials' [line 103]: section 'None' already exists

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jbpratt/.local/bin/stsauth", line 8, in <module>
    sys.exit(cli())
  File "/home/jbpratt/.local/lib/python3.8/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/jbpratt/.local/lib/python3.8/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/sts_auth/cli.py", line 129, in authenticate
    token = sts_auth.fetch_aws_sts_token(role_arn, principal_arn, saml_response.assertion, aws_profile=profile)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/sts_auth/stsauth.py", line 240, in fetch_aws_sts_token
    raise e
  File "/home/jbpratt/.local/lib/python3.8/site-packages/sts_auth/stsauth.py", line 233, in fetch_aws_sts_token
    session = boto3.Session(profile_name=aws_profile)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/boto3/session.py", line 80, in __init__
    self._setup_loader()
  File "/home/jbpratt/.local/lib/python3.8/site-packages/boto3/session.py", line 120, in _setup_loader
    self._loader = self._session.get_component('data_loader')
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 682, in get_component
    return self._components.get_component(name)
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 919, in get_component
    self._components[name] = factory()
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 155, in <lambda>
    lambda:  create_loader(self.get_config_variable('data_path')))
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 237, in get_config_variable
    return self.get_component('config_store').get_config_variable(
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/configprovider.py", line 293, in get_config_variable
    return provider.provide()
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/configprovider.py", line 390, in provide
    value = provider.provide()
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/configprovider.py", line 451, in provide
    scoped_config = self._session.get_scoped_config()
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 327, in get_scoped_config
    profile_map = self._build_profile_map()
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 223, in _build_profile_map
    self._profile_map = self.full_config['profiles']
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/session.py", line 364, in full_config
    cred_profiles = botocore.configloader.raw_config_parse(
  File "/home/jbpratt/.local/lib/python3.8/site-packages/botocore/configloader.py", line 151, in raw_config_parse
    raise botocore.exceptions.ConfigParseError(
botocore.exceptions.ConfigParseError: Unable to parse config file: /home/jbpratt/.aws/credentials
```

```
~/work tail ~/.aws/credentials
[None]

[None]
```

**Comment by jbpratt78** [comment]
Fixed with #58

**Comment by jbpratt** [comment]
Closed with #60
